doc/usecases.md

```diff
@@ -0,0 +1,188 @@
+#Layer2
+
+Usage can refer to this [doc](layer2.md)
+
+## Prerequisite
+
+The cluster should have at least two nodes, nodeA and nodeB.
+
+## Test endpoint update
+
+- create service test-svc which have one endpoint on nodeA.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mylbapp
+  labels:
+    app: mylbapp
+spec:
+  containers:
+  - name: helloworld
+    image: karthequian/helloworld
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/hostname: nodeA
+---
+kind: Service
+apiVersion: v1
+metadata:
+    name:  test-svc
+    annotations:
+        lb.kubesphere.io/v1alpha1: porter
+        #eip.porter.kubesphere.io/v1alpha1: 1.1.1.1 
+        #protocol.porter.kubesphere.io/v1alpha1: layer2 
+spec:
+    selector:
+        app:  mylbapp
+    type:  LoadBalancer 
+    ports:
+      - name:  http
+        port:  80
+        targetPort:  80
+```
+
+- exec 'wget ${eip}' can successfully access to test-svc
+- exec 'ip neigh  show to ${eip} dev ${nic}', we can find lladdr is nodeA's Mac
+- delete the endpoint which on nodeA, and create endpoint on nodeB
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mylbapp
+  labels:
+    app: mylbapp
+spec:
+  containers:
+  - name: helloworld
+    image: karthequian/helloworld
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/hostname: nodeB
+```
+- exec 'wget ${eip}' can successfully access to test-svc
+- exec 'ip neigh  show to ${eip} dev ${nic}', we can find lladdr is nodeB's Mac
+
+# BGP
+
+Usage can refer to this [doc](simulate_with_bird.md)
```

Comment:
[BGP guide](simulate_with_bird.md) demonstrates how to install Porter and use BGP on Kubernetes.